examples/todo-app-tutorial/src/todo-app.ts

```diff
@@ -32,7 +32,10 @@ const template = html<TodoApp>`
                     <span class="list-text ${x => (x.done ? "done" : "")}">
                         ${x => x.description}
                     </span>
-                    <fast-button @click=${(x, c) => c.parent.removeTodo(x)}>
+                    <fast-button
+                        @click=${(x, c) => c.parent.removeTodo(x)}
+                        aria-label="${x => x.ariaLabel}"
```

Comment:
Since the label isn't going to change, I'd be inclined to remove the attr and just hard code the value in the template. Thoughts?

Reply:
Yeah, if it is not going to change, then I am ok with just hard coding the value.